Habitica/src/com/habitrpg/android/habitica/MainActivity.java

```diff
@@ -739,4 +745,31 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
 
         return super.onKeyUp(keyCode, event);
     }
+
+    private void scheduleReminder(Context context) {
+
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(this);
+        String timeval = prefs.getString("reminder_time", "19:00");
+
+        if (timeval == null) timeval = "19:00";
```

Comment:
Currently this sets the reminder to 19:00 even if it's turned off. It should check the preference "use_reminder" first if it's set. and only set the reminder if that is active.